### DISCUSSION TITLE
Waveshare HDMI USB Touchscreen

### DISCUSSION THREAD

**Discussion by rvxfahim** [ImpulseAdventure/GUIslice#366] [Q&A]
Hi @ImpulseAdventure,

I am pretty used to guislice now for developing projects for arduino. Now I am thinking about doing something with Raspberry Pi.
I wanted to know if I can use HDMI touchscreens like some made by waveshare (https://www.waveshare.com/wiki/7inch_HDMI_LCD_(C)) which has a usb cable with the display that gives capacitive touch support in the Raspberry Pi OS without any need for extra driver installation. Can I use these displays instead of PiTFT SPI displays to make touch programs ?

**Comment by ImpulseAdventure** [comment]
Hi @rvxfahim -- I have not yet tested these displays with GUIslice but it has been on my list to order a similar one and confirm it works. If you already have the hardware, and would like me to help you get it going, let me know.

**Reply by GeoffMP** [reply]
Hi, I am also looking at the waveshare hdmi display. In my case I am attempting to drive the 11.9" display. I am a real beginner and have managed to configure a pi4b to display the desktop...now looking at using guislice to build my own apps UI. Did you ever get to testing a waveshare display?
